WalletWasabi.Backend/Global.cs

```diff
@@ -90,7 +93,7 @@ public async Task InitializeAsync(Config config, CoordinatorRoundConfig roundCon
 			var indexFilePath = Path.Combine(indexBuilderServiceDir, $"Index{RpcClient.Network}.dat");
 			var blockNotifier = HostedServices.FirstOrDefault<BlockNotifier>();
 			IndexBuilderService = new IndexBuilderService(RpcClient, blockNotifier, indexFilePath);
-			Coordinator = new Coordinator(RpcClient.Network, blockNotifier, Path.Combine(DataDir, "CcjCoordinator"), RpcClient, roundConfig, () =>  pushService.SendNotificationsAsync(false));
+			Coordinator = new Coordinator(RpcClient.Network, blockNotifier, Path.Combine(DataDir, "CcjCoordinator"), RpcClient, roundConfig, () => _ = serviceProvider.GetRequiredService<SendPushService>().SendNotificationsAsync(false));
```

Comment:
Can make this cleaner by introducing an event (and  moving the coordinator into the DI)